package.json

```diff
@@ -65,12 +65,12 @@
     "codecov": "^3.6.5",
     "espower-typescript": "^9.0.2",
     "gts": "^1.1.2",
-    "mocha": "^7.0.0",
+    "mocha": "^7.1.1",
     "ncp": "^2.0.0",
     "rimraf": "^3.0.2",
-    "typescript": "~3.7.5"
+    "typescript": "~3.8.3"
   },
   "engines": {
-    "node": ">=8.13.0"
+    "node": ">=10.0.0"
```

Comment:
Can we do the same for all the baselines' and template `package.json` in the same PR? Drop means drop! (for the generated libraries too)

Reply:
Great suggestion! done!

Reply:
Also updated `typescript` version for the generate client library! :)